app/main.py

```diff
@@ -1,8 +1,47 @@
+from typing import Any
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_mark: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance = distance_from_city_center
+        self.clean_mark = clean_mark
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Any) -> float:
+        if self.clean_mark <= car.clean_mark:
+            return 0.0
+        temp = car.comfort_class * self.average_rating
+        temp = temp * (self.clean_mark - car.clean_mark)
+        return round(temp / self.distance, 1)
+
+    def wash_single_car(self, car: Any) -> None:
+        if self.clean_mark <= car.clean_mark:
+            return
+        car.clean_mark = self.clean_mark
+
+    def rate_service(self, rate: int) -> None:
+        self.average_rating *= self.count_of_ratings
```

Comment:
Explain why do you need this multiplication.

Reply:
Simplified it in new commit